src/config.rs

```diff
@@ -0,0 +1,83 @@
+use crate::args::Args;
+use crate::errors::*;
+use serde::Deserialize;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+const DEFAULT_URL: &str = "https://security.archlinux.org/all.json";
+
+pub struct Config {
+    pub source: String,
+    pub proxy: Option<String>,
+}
+
+impl Config {
+    pub fn load(args: &Args) -> Result<Self> {
+        let mut merged = ConfigFile::default();
+
+        let mut configs = vec![PathBuf::from("/etc/arch-audit/arch-audit.toml")];
+        if let Some(dir) = dirs_next::config_dir() {
+            let path = dir.join("arch-audit/arch-audit.toml");
```

Comment:
I'm confused, are we now using `settings.toml` inside the directory as the contrib example files is named or `arch-audit.toml`? :D i guess settings is better suited as arch-audit is already the context from within the namespace.

Reply:
right, changed it back to settings.toml